src/views/utils.js

```diff
@@ -46,3 +49,50 @@ export function getDefaultInitialViewState(loader, viewSize, zoomBackOff = 0) {
   };
   return loaderInitialViewState;
 }
+
+/**
+ * Creates the layers for viewing an image in detail.
+ * @param {string} id The identifier of the view.
+ * @param {Object} props The layer properties.
+ * @returns {Array} An array of layers.
+ */
+export function getImageLayers(id, props) {
+  const {
+    loader,
+    loaderSelection,
+    newLoaderSelection,
+    onViewportLoad,
+    transitionFields
+  } = props;
```

Comment:
Can we add something like this and then use `layerProps` instead of `props` below for the `MultiscaleImageLayer`?
```javascript
const layerProps = {}
Object.keys(props).map(k => {
  if(k !== 'transitionFields' || k !== 'newLoaderSelection') {
    layerProps[k] = props[k]
  }
})
```
I just want to try to avoid unnecessary props to the layers.

Reply:
I did that via destructuring instead.

Reply:
Much cleaner, yes, thanks.